### DISCUSSION TITLE
Impossible to sync

### DISCUSSION THREAD

**Discussion by Lowlee3D** [opengisch/QField#4074] [Bugs]
Hi,
I have a big problem:
I modified a project in my phone and I uploaded it, but when I try to sync from my pc the project has no changes.
How I can recover the modified project from my phone?

Thank you all

**Comment by BoswachterMarc** [comment]
You can send a compressed folder from QField or manually, by cable. See https://docs.qfield.org/get-started/storage/

**Reply by Lowlee3D** [reply]
I can't find any folder in my phone, qfield folders are empty

**Comment by ZsanettMed** [comment]
Hi @Lowlee3D 
Could you provide us more information about your issue (username, project name and/or any error message)? 

Thank you and best regards, 
Zsanett

**Reply by Lowlee3D** [reply]
My user is lowlee, I read some error messages when I was offline, but I read it was refered to the google hybrid layer.

I would like to recover the project from my phone and delete the cloud one. Is it possible?

Thank you

**Reply by ZsanettMed** [reply]
Did you try the "Sent to..." functionality? https://docs.qfield.org/how-to/projects/?h=send#send-to

**Reply by Lowlee3D** [reply]
Yes, but the folders "imported projects" and "qfield" in the phone are empty.
